src/write.c

```diff
@@ -771,6 +771,18 @@ static avifResult avifEncoderWriteExtendedColorProperties(avifRWStream * dedupSt
     return AVIF_RESULT_OK;
 }
 
+static avifResult avifEncoderWriteHandlerBox(avifRWStream * s, const char handler_type[4])
```

Comment:
Nit: The function name should start with `avifRWStream` (the type of the first/main parameter) intead of `avifEncoder`.